portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1203,6 +1247,16 @@ definition {
 			value1 = "${fieldValue}");
 	}
 
+	macro viewDisabledSelectOption {
+		Click(
+			key_fieldName = "${fieldName}",
+			locator1 = "FormFields#SELECT_FIELD");
+
+		AssertElementPresent(
+			key_text = "${optionValue}",
+			locator1 = "FormFields#ANY_DISABLED_DROPDOWN");
```

Comment:
@leticiachiu, can you please just replace the word `DROPDOWN` here too? This is the last place remaining this word in this pull.

Once you perform this SF, please feel free to trigger `ci:forward`. :wink: 

Thanks.